src/ui/components/shared/SharingModal/PrivacyDropdown.tsx

```diff
@@ -157,8 +157,18 @@ export default function PrivacyDropdown({ recording }: { recording: Recording })
     );
   }
 
+  const preloadIcons = () => (
+    <div style={{ display: "none" }}>
+      <MaterialIcon>globe</MaterialIcon>
+      <MaterialIcon>group</MaterialIcon>
+      <MaterialIcon>lock</MaterialIcon>
+      <MaterialIcon>expand_more</MaterialIcon>
+    </div>
+  );
+
   return (
     <>
+      {preloadIcons()}
```

Comment:
💬 This one, on the other hand, _will_ do something, because it's included in the JSX return structure.

:hand: That said, in both cases, `preloadIcons` doesn't need to be a _function_.  It could just be a direct assignment to a variable, ie, `const preloadIcons = <div>Stuff Here</div>`

Reply:
I did not know that, thank you!

Reply:
PRs like this should start going through normal PR review process. Mark, Holger, and I review each other's changes as well. It's just got to have a second set of eyes on stuff that impacts production code.